coops-ui/src/components/Dashboard/DirectorListSm.vue

```diff
@@ -1,71 +1,114 @@
 <template>
-  <ul class="list list-container">
-    <li class="list-item" v-for="director in directors" v-bind:key="director.id">
-      <v-avatar color="primary" size="25">
-        <span class="white--text small">{{ director.officer.firstName.substring(0,1)}}</span>
-      </v-avatar>
-      <div class="director-info">
+  <v-expansion-panels class="list list-container" :accordion=true :multiple=true>
+    <v-expansion-panel class="address-panel" v-for="director in directors" v-bind:key="director.id">
+      <v-expansion-panel-header>
+        <v-avatar color="primary" size="25">
+        <span class="">{{ director.officer.firstName.substring(0,1)}}</span>
```

Comment:
Indent please.

Reply:
Good catch, thanks!

Reply:
If the linter doesn't catch this, we should add a rule.